lldb/test/Shell/helper/toolchain.py

```diff
@@ -149,7 +149,10 @@ def use_support_substitutions(config):
             sdk_path = lit.util.to_string(out)
             llvm_config.lit_config.note("using SDKROOT: %r" % sdk_path)
             host_flags += ["-isysroot", sdk_path]
-    elif sys.platform != "win32":
+    elif sys.platform == "win32":
+        # Required in SwiftREPL tests
+        llvm_config.with_system_environment("SDKROOT")
+    else:
```

Comment:
I guess instead of `lit.cfg.py` we better propagate the `SDKROOT` environment variable here, because that where it's done for macOS.

Reply:
what if `SDKROOT` isn't set in the environment, does this silently fail, or not do anything? Perhaps it could warn if SDKROOT isn't set.

Reply:
Good question, let me check

Reply:
It fails silently. Let me add a patch for it tomorrow.